src/components/network/NetworkParamsEdit.vue

```diff
@@ -363,7 +363,7 @@
               item-value="value"
               item-text="label"
               label="Connection rule"
-              class="ml-1 px-1"
+              class="ml-1 px-1 py-1"
```

Comment:
you can take the class pa-1 instead of px-1 and py-1.

```suggestion
              class="ml-1 pa-1"
```